libraries/ESP32/examples/AnalogOut/LEDCGammaFade/LEDCGammaFade.ino

```diff
@@ -0,0 +1,117 @@
+/* LEDC Gamma Curve Fade Arduino Example
+
+   This example demonstrates gamma curve fading on ESP32 variants that support it.
+   Gamma correction makes LED brightness changes appear more gradual and natural
+   to human eyes compared to linear fading.
+
+   Two methods are supported:
+   1. Using a pre-computed Gamma Look-Up Table (LUT) for better performance
+   2. Using mathematical gamma correction with a gamma factor
+
+   Supported chips: ESP32-C6, ESP32-C5, ESP32-H2, ESP32-P4 and future chips with Gamma Fade support
+
+   Created by Jan Procházka (https://github.com/P-R-O-C-H-Y/)
+*/
+
+// use 12 bit precision for LEDC timer
+#define LEDC_TIMER_12_BIT 12
+
+// use 5000 Hz as a LEDC base frequency
+#define LEDC_BASE_FREQ 5000
+
+// define starting duty, target duty and maximum fade time
+#define LEDC_START_DUTY  (0)
+#define LEDC_TARGET_DUTY (4095)
+#define LEDC_FADE_TIME   (2000)
+
+// gamma factor for mathematical calculation
+#define LEDC_GAMMA_FACTOR (2.6) 
+
+// use gamma LUT for better performance instead of mathematical calculation (gamma factor)
+#define USE_GAMMA_LUT 1 
+
+// fade LED pins
+const uint8_t ledPinR = 4;
+const uint8_t ledPinG = 5;
+const uint8_t ledPinB = 6;
+
+uint8_t fade_ended = 0;  // status of LED gamma fade
+bool fade_in = true;
+
+#ifdef USE_GAMMA_LUT
+// Custom Gamma LUT demonstration with only 10 steps (the default built-in Gamma lut is 101 steps)
```

Comment:
it says that this is demonstration with 10 steps, yet the array has 101 items. What is true? Please clarify this comment better

Reply:
Old comment before I changed the code, will fix